e2e/content-services/permissions-component.e2e.ts

```diff
@@ -50,20 +48,25 @@ describe('Permissions Component', function () {
     };
 
     let groupId;
+    let alfrescoJsApi = new AlfrescoApi({
+        provider: 'ECM',
+        hostEcm: TestConfig.adf.url
+    });
+
+    acsUser = new AcsUserModel();
+
+    acsUser2 = new AcsUserModel();
+    const duplicateUserPermissionMessage = 'One or more of the permissions you have set is already present : authority -> ' + acsUser2.getId() + ' / role -> Contributor';
```

Comment:
do we need to create a const if we check it just once?

Reply:
it's for better readability.